### DISCUSSION TITLE
我想换新电脑，dell g15安装ubuntu打游戏体验如何？

### DISCUSSION THREAD

**Discussion by izumi-sora** [Gamuxorg/bbs#308] [建议和闲聊]
游戏名称：网站上的游戏

操作系统：*（uname -a）* G15 游戏本  https://www.dell.com/zh-cn/shop/%E7%AC%94%E8%AE%B0%E6%9C%AC/g15-%E7%AC%AC12%E4%BB%A3intel%E7%89%88-%E6%B8%B8%E6%88%8F%E6%9C%AC/spd/g-series-15-5520-laptop/g15%205520-d1646b

bug描述：*（具体说说发生了什么）*  打游戏最好不卡，加载尽量流畅，不要浪费电脑资源，尽量便宜一点，预算最大有1万人民币

**Comment by shouhuanxiaoji** [comment]
这个配置应该可以玩绝大多数游戏，但为啥不买台式机？

**Comment by izumi-sora** [comment]
基本上不在家，台式机搬起来太费劲了
